shared/src/hooks/useFetchV2VaultData.ts

```diff
@@ -116,7 +117,12 @@ const useFetchV2VaultData = (): V2VaultData => {
           _withdrawals,
         ] = await Promise.all(
           // Default to 0 when error
-          promises.map((p) => p.catch((e) => BigNumber.from(0)))
+          promises.map((p) =>
+            p.catch((e) => {
+              console.log(e);
```

Comment:
Do we want to leave this console.log in?

Reply:
I was running into an issue with pulling the vault cap and the proxy will reject the call if you're connected to the deployer account.  If we had a console.log, it would have taken no time to find the problem.  But we were suppressing the error and I had to go down the rabbit hole to debug the issue.